src/frontend/src/app/assets/assets.component.ts

```diff
@@ -8,14 +8,14 @@ import { AssetsService, AlertService } from '../services/index';
 })
 export class AssetsComponent implements OnInit {
 
-  selectedAsset: Object = 'Select' // Selected asset object (asset_coded, asset_count)
-  asset: Object;
-  limit: number = 20;        
-  offset: number = 0;
```

Comment:
as above, type hint

Reply:
Lint check giving error **[tslint] Type number trivially inferred from a number literal, remove type annotation (no-inferrable-types)** with primitive types *number, string, boolean*  because of **no-inferrable-types = true** in lint.json.
Either use **Number** wrapper class on nothing or change **no-inferrable-types = false**  to remove the lint error.

Reply:
I think its fine then; I think we should consider now. when you have declaration or func arg def only then use type hint; else it will be auto checked with assignment.